### PR TITLE
Support Channel access token v2.1

### PR DESCRIPTION
**Pull request by kazuki-ma**
Integratio test (with my token) result
===

image

### CODE REVIEW

File: line-bot-api-client/src/integrationTest/java/com/linecorp/bot/client/LineOAuthClientIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import com.linecorp.bot.model.oauth.IssueChannelAccessTokenResponse;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.jackson.io.JacksonSerializer;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class LineOAuthClientIntegrationTest {
+    public static final URL TEST_RESOURCE = ClassLoader.getSystemResource("integration_test_settings.yml");
```
Comment (imasahiro) [inline line-bot-api-client/src/integrationTest/java/com/linecorp/bot/client/LineOAuthClientIntegrationTest.java:49]: Could be `private`?

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java [hunk 2]
```diff
@@ -35,6 +37,32 @@ static LineOAuthClientBuilder builder() {
...
+     *
+     * <p>You can issue up to 30 tokens.
+     * If you reach the maximum limit, additional requests of issuing channel access tokens are blocked.
+     *
+     * @param clientAssertion A JSON Web Token the client needs to create and sign with the private key created
+     *                        when issuing an assertion signing key.
+     * @see <a href="https://developers.line.biz/en/reference/messaging-api/#issue-channel-access-token-v2-1">Issue channel access token v2.1</a>
+     */
+    CompletableFuture<IssueChannelAccessTokenResponse> issueChannelTokenByJWT(
+            @Field("client_assertion") String clientAssertion);
```
Comment (imasahiro) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java:51]: We don't need to add `@Field` annotation here, isn't it?

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java [hunk 3]
```diff
@@ -35,6 +37,32 @@ static LineOAuthClientBuilder builder() {
...
+    /**
+     * Revokes a channel access token.
+     *
+     * @param clientId     Channel ID
+     * @param clientSecret Channel Secret
+     * @param accessToken  Channel access token
+     * @see <a href="https://developers.line.biz/en/reference/messaging-api/#revoke-channel-access-token-v2-1">Revoke channel access token v2.1</a>
+     */
+    CompletableFuture<Void> revokeChannelTokenByJWT(
+            @Field("client_id") String clientId,
```
Comment (imasahiro) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java:62]: ditto

File: line-bot-api-client/src/integrationTest/java/com/linecorp/bot/client/IntegrationTestSettingsLoader.java [hunk 4]
```diff
@@ -26,7 +26,7 @@
 import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
 
 public class IntegrationTestSettingsLoader {
-    public static final URL TEST_RESOURCE = ClassLoader.getSystemResource("integration_test_settings.yml");
+    private static final URL TEST_RESOURCE = ClassLoader.getSystemResource("integration_test_settings.yml");
```
Comment (kazuki-ma) [inline line-bot-api-client/src/integrationTest/java/com/linecorp/bot/client/IntegrationTestSettingsLoader.java:29]: Fixed too from comment https://github.com/line/line-bot-sdk-java/pull/508#discussion_r423422635

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java (hunk 2 above)
Comment (kazuki-ma) [reply line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java]: 🙏 done.

**Comment by kazuki-ma** [comment]
Thank you! Updated~

**Pull request merged**